pyspark_ai/prompt.py

```diff
@@ -98,43 +155,27 @@
 Observation: OK
 Thought: I now know the final answer.
 Final Answer: SELECT COUNT(`Student`) FROM `spark_ai_temp_view_12qcl3` WHERE `Birthday` = '01-01-2006'"""
-    """QUESTION: Given a Spark temp view `spark_ai_temp_view_wl2sdf` with the following columns:
-```
-PassengerId INT
-Survived INT
-Pclass INT
-Name STRING
-Sex STRING
-Age DOUBLE
-SibSp INT
-Parch INT
-Ticket STRING
-Fare DOUBLE
-Cabin STRING
-Embarked STRING
-```
-Write a Spark SQL query to retrieve from view `spark_ai_temp_view_wl2sdf`: What's the name of the oldest survived passenger?
-Thought: I will query the Name and Age columns, filtering by Survived and ordering by Age in descending order.
-Action: query_validation
-Action Input: SELECT Name, Age FROM spark_ai_temp_view_wl2sdf WHERE Survived = 1 ORDER BY Age DESC LIMIT 1
-Observation: OK
-Thought:I now know the final answer.
-Final Answer: SELECT Name, Age FROM spark_ai_temp_view_wl2sdf WHERE Survived = 1 ORDER BY Age DESC LIMIT 1"""
+
+SPARK_SQL_EXAMPLES_NO_VECTOR_SEARCH = [
+    spark_sql_no_vector_example_1,
+    spark_sql_shared_example_1,
+    spark_sql_no_vector_example_2,
+    spark_sql_shared_example_2,
 ]
 
-SPARK_SQL_SUFFIX = """\nQuestion: Given a Spark temp view `{view_name}` {comment}.
-The dataframe contains the column names and types in this format:
-column_name: type.
-It's very important to ONLY use the verbatim column names in your resulting SQL query.
+SPARK_SQL_EXAMPLES_VECTOR_SEARCH = [
+    spark_sql_vector_example_1,
```

Comment:
Let's abstract the common parts of 
* spark_sql_vector_example_1 & spark_sql_no_vector_example_1
* spark_sql_no_vector_example_2 & spark_sql_vector_example_2

into variables.